esm-utils.mjs

```diff
@@ -3,32 +3,38 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
+'use strict';
+// TODO(esmodules): rename to .js when root is type: module
 
 import module from 'module';
 import url from 'url';
 import path from 'path';
 
-const require = module.createRequire(import.meta.url);
-
 /**
  * Commonjs equivalent of `require.resolve`.
  * @param {string} packageName
  */
 function resolveModulePath(packageName) {
+  const require = module.createRequire(import.meta.url);
   return require.resolve(packageName);
 }
 
 /**
  * @param {ImportMeta} importMeta
  */
-function createCommonjsRefs(importMeta) {
-  const require = module.createRequire(importMeta.url);
-  const filename = url.fileURLToPath(importMeta.url);
-  const dirname = path.dirname(filename);
-  return {require, __filename: filename, __dirname: dirname};
+function getModuleName(importMeta) {
```

Comment:
Is there a situation where this doesn't return a file path? If not could this be `getModulePath` instead?